app/models/offer.rb

```diff
@@ -3,6 +3,4 @@ class Offer < ApplicationRecord
 
   validates :title, presence: true
   validates :description, presence: true
-  validates :start_time, presence: true
-  validates :end_time, presence: true
```

Comment:
delete this because it does not make sense in regard to offers...we will use it at bookings